compute_tools/src/bin/compute_ctl.rs

```diff
@@ -78,6 +81,16 @@ fn main() -> Result<()> {
                 let path = Path::new(sp);
                 let file = File::open(path)?;
                 serde_json::from_reader(file)?
+            } else if let Some(id) = compute_id {
+                if let Some(cp_base) = control_plane_uri {
+                    let cp_uri = cp_base.to_owned() + "/management/api/v1/" + id + "/spec";
+                    reqwest::blocking::get(cp_uri)?
+                        // TODO: fetch a jwt token from the environment
+                        // .header("Authorization", "JWT TOKEN")
```

Comment:
Maybe just implement it right away instead of putting a TODO? Seems to be pretty trivial